### DISCUSSION TITLE
the wizard is always launched

### DISCUSSION THREAD

**Discussion by abyrnoradan** [WinEdt-Team/WinEdt-Support#67] [1.1 General Help and Discussion]
Each time i launch winedt i have this exasperate wizard pop up. 
Of course I uncheck the box "show this wizard next time" but as usual with me nothing happened !
I continue to have this popup.
What can i do ?

**Comment by Alex-Simonic** [comment]
This is not a common behavior experienced by others. At the bottom of the Wizard, you will see `Local Folder (%b)`. Can you tell me what is there (use copy and paste). Are you allowed to write in this folder without elevated privileges?

On a related note: did you install WinEdt for all users (inside a Program Files) AND disable the user profile creation during the installation?
